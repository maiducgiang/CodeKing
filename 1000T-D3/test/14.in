54
19819 53059 13778 64991
47086 55569 39545 56745
41064 41748 84605 83892
88496 87389 43582 32470
21165 42410 90522 58398
31218 18861 38095 38279
24641 32783 30727 11843
13662 50467 7409 50835
15469 18749 45903 84733
18040 27519 31650 65933
52417 90283 83079 70394
2750 72404 82183 13822
5181 67773 82674 52764
97190 82179 50943 43341
34059 47898 76189 43561
30485 80579 52292 75542
91773 84322 8485 95235
28230 30865 7696 86034
43588 58644 31611 2045
48844 96161 19919 91034
92456 19047 60144 62444
45229 82508 91881 92215
61729 39722 75942 64702
36659 70524 4983 57986
30591 96652 49522 96364
36326 85438 9250 77405
81844 81905 83909 64586
64541 96616 35339 11822
7566 15965 2695 25966
62435 20750 28915 86151
9257 55885 21260 51956
13814 36975 85931 35670
66371 59455 33693 79796
82921 69931 34892 53678
59537 11630 65571 34909
49771 58158 29917 35447
36583 73748 87469 72737
29042 57341 54298 4550
83605 90323 89523 50529
94429 15499 98239 17482
11988 14878 99455 60077
48963 96283 26136 13273
73673 24914 65113 46847
89926 61828 46843 47691
5429 41819 86264 72389
24754 35874 45512 37408
15047 20850 91607 57177
56738 68979 3922 42644
88251 27319 94752 89308
57116 84552 43531 92154
93929 15610 88883 56489
97576 19813 21570 73088
81872 27202 55852 28662
57762 5253 97125 8259
