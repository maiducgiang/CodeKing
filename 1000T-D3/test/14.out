1004113084
3007355305
3502810908
2885766104
3159133560
1192720134
542002597
475131761
1461296617
1441942350
6835711712
5762385182
5448101292
4326425412
2538023670
4922413318
1494166075
454795120
565072676
2888354595
5604481764
7596024034
4241348438
534092889
6219343966
1126317000
5291095033
1896502127
63416045
2405294275
1384918920
2482993271
4239455418
4207003298
3547301567
2015146516
5911723056
1668720202
4825892559
8024957788
1007628026
1886641419
4341990925
4264833762
3532146841
1432091072
1391937740
493062074
8213234116
6166949708
7843020967
2532080730
3833092564
5143321152
